platform-api/src/routers/discord.py

```
@@ -0,0 +1,96 @@
+
+    window_seconds = 60
+    key = f"rate:discord:{channel}"
+    raw = await kv.get(key)
+    now = int(time.time())
+
+    if raw is None:
+        # First write: initialize count and timestamp
+        data = {"count": 1, "started_at": now}
+        await kv.put(key, json.dumps(data), {"expirationTtl": window_seconds})
+        return False
+    else:
```

Ruff (0.15.11)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)